manifests/a/AntibodySoftware/WizTree/4.13/AntibodySoftware.WizTree.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Automatically updated by the winget bot at 2023/Mar/16
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AntibodySoftware.WizTree
+PackageVersion: "4.13"
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /NORESTART
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: WizTree v4.12
+ReleaseDate: 2022-12-01
+Installers:
+- Architecture: x64
+  InstallerUrl: https://diskanalyzer.com/files/wiztree_4_12_setup.exe

```

Comment:
URL Redirects to 4.13 anyways, so doesn't need to be changed. Leaving as a note for time travellers